test/patterns/prime generator/prime.stdin

```diff
@@ -3,4 +3,3 @@
 10
 5
 50
```

Comment:
Hmm.  After doing a performance run, I have changed my mind.  Would you mind using a larger problem size to verify that this is working?  This completes in 1.5e-5 on my machine and that is just too fast for us to get useful timing information out of it (as at that speed, most of the overhead will be task start up).